Get rid of allow + -ing (#5045)
DevExpress/devextreme-documentation: api-reference/_hidden/dxDataGridColumn/cellTemplate.md

The cell's raw value.

##### field(cellInfo.watch): function()

> Allows to track a variable and performing actions when it changes. Applies when [repaintChangesOnly](/api-reference/10%20UI%20Components/GridBase/1%20Configuration/repaintChangesOnly.md '{basewidgetpath}/Configuration/#repaintChangesOnly') is **true**.       

Reviewer: ```suggestion
Allows you to track a variable and respond to value changes. Applies when [repaintChangesOnly](/api-reference/10%20UI%20Components/GridBase/1%20Configuration/repaintChangesOnly.md '{basewidgetpath}/Configuration/#repaintChangesOnly') is **true**.       
```
